pkg/dockerregistry/server/auth.go

```diff
@@ -125,14 +125,9 @@ func (ac *AccessController) Authorized(ctx context.Context, accessRecords ...reg
 		return nil, ac.wrapErr(err)
```

Comment:
add a test for a basic auth header of base64("foo:") to make sure this is not allowed